wiki/FitNesseRoot/HsacExamples/SlimTests/HttpTests/XmlHttpTestResponseSchemaValidation.wiki

```diff
@@ -0,0 +1,54 @@
+---
+Test
+---
+|script           |mock xml server setup                             |
+|add response file|http://files/xmlTestExamples/valid_shiporder.xml  |
+|add response file|http://files/xmlTestExamples/invalid_shiporder.xml|
+|$url=            |get mock server url                               |
+
+!*> XSD String definition
+!define xsdSchema { {{{<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
The test might be more maintainable if you use FileFixture in wiki page explicitly to load the file into a Slim symbol, and pass that to the xml http fixture method,  instead of repeating the xsd contents here.